### ISSUE TITLE
White dots showing up in the image

### ISSUE THREAD

**Issue by W-OK-E** [eezkni/FDL#5]
Thank you for sharing your amazing work, but I had some confusing results when training with FDL loss.

So I am training a UNET for denoising, and I am using a mix of SSIM and FDL loss, but the denoised image ends up having artifacts as shown:

![Image](https://github.com/user-attachments/assets/40dc5dfc-2e66-4432-8bfa-596523b4729a)

The white dots shown are the artifacts. Is there any way to mitigate that?

**Comment by Chtholly17** [comment]
Hi. 

For denoising task, it is suggested to employ pixel-level loss function such as L1 norm. Denosing requrie the model to remove the local noise, which is a kind of **high-frequency** degradation. In my experience, using point-to-point supervision can help the model to remove such degradation most effectively. If there is misalignment within your training data, you can try to combine FDL loss with point-to-point loss such as L1.

**Comment by W-OK-E** [comment]
Thank You for the insights! Grateful !
